internal/rootcoord/proxy_client_manager.go

```diff
@@ -113,30 +113,22 @@ func (p *proxyClientManager) InvalidateCollectionMetaCache(ctx context.Context,
 		return
 	}
 
-	for k, f := range p.proxyClient {
-		err := func() error {
-			defer func() {
-				if err := recover(); err != nil {
-					log.Debug("call InvalidateCollectionMetaCache panic", zap.Int64("proxy id", k), zap.Any("msg", err))
-				}
-
-			}()
+	wg := &sync.WaitGroup{}
```

Comment:
why there is no error returned?
please check on concurrent/Future on how to handle multi request

Reply:
fixed